modules/quickstart/pages/network-quickstart.adoc

```diff
@@ -131,26 +139,189 @@ To check your connection to the {IC} network:
++
+[source,bash]
+----
+dfx identity get-principal
+----
++
+This command displays output similar to the following:
++
+....
+tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe
+....
+. Get the account identifier for your developer identity by running the following command:
++
+[source,bash]
+----
+dfx ledger account-id
+----
++
+This command displays the ledger account identifier associated with your developer identity.
+For example, you should see output similar to the following:
++
+....
+03e3d86f29a069c6f2c5c48e01bc084e4ea18ad02b0eec8fccadf4487183c223
+....
+. Check your account balance by running the following command:
++
+[source,bash]
+----
+dfx ledger --network ic balance
+----
+This command displays the ICP token balance from the ledger account.
+For example, you should see output similar to the following:
++
+....
+100.00000000 ICP
+....
+
+[[convert-icp]]
+== Convert ICP tokens to cycles
+
+Now that you have confirmed your account information and current ICP token balance, you can convert some of those ICP tokens to cycles and move them into a cycles wallet.
+
+To transfer ICP tokens to create a cycles wallet:
+
+. Create a new canister with cycles by transferring ICP tokens from your ledger account by running a command similar to the following:
++
+[source,bash]
+----
+dfx ledger --network ic create-canister --amount <icp-tokens> --controller <principal-identifier>
+----
++
+This command converts the number of ICP tokens you specify for the `+--amount+` argument into cycles, and associates the cycles with a new canister identifier controlled by the principal you specify using the `+--controller+` argument.
++
+For example, the following command converts 1.25 ICP tokens into cycles and specifies the principal identifier for the default identity as the controller of the new canister:
++
+....
+dfx ledger --network ic create-canister --amount 1.25 --controller tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe
+....
++
+If the transaction is successful, the ledger records the event and you should see output similar to the following:
++
+....
+Transfer sent at BlockHeight: 20
+Canister created with id: "53zcu-tiaaa-aaaaa-qaaba-cai"
+....
+. Install the cycles wallet code in the newly-created canister placeholder by running a command similar to the following:
++
+[source,bash]
+----
+dfx identity --network ic deploy-wallet <canister-identifer>
+----
++
+For example:
++
+....
+dfx identity --network ic deploy-wallet 53zcu-tiaaa-aaaaa-qaaba-cai
+....
++
+This command displays output similar to the following:
++
+....
+Creating a wallet canister on the ic network.
+The wallet canister on the "ic" network for user "default" is "53zcu-tiaaa-aaaaa-qaaba-cai"
+....
+
+== Validate your cycles wallet
+
+After you convert ICP tokens to cycles, you might want to validate the cycles wallet canister and your current cycles balance.
```

Comment:
NIT: delete "you might want" altogether or replace with "...you **can** validate"

Reply:
yeah, that's better.